src/accelerate/accelerator.py

```diff
@@ -1704,7 +1704,7 @@ def save_state(self, output_dir: str = None):
         self.project_configuration.iteration += 1
         return save_location
 
-    def load_state(self, input_dir: str):
+    def load_state(self, input_dir: str, **model_load_kwargs):
```

Comment:
```suggestion
    def load_state(self, input_dir: str, **load_func_kwargs):
```

I think we should rename this to `load_func_kwargs` which is a bit more explicit in what these are actually going towards.

Reply:
hello, but `load_func` can mean optimizer/scheduler loading too, and hence chose this name.

Reply:
True, `load_model_func_kwargs` perhaps then as a medium

Reply:
Done